upgrade/1.0.1/README.md

```diff
@@ -41,6 +41,13 @@
 
     See [Check for Latest Workarounds and Documentation Updates](../../update_product_stream/index.md#workarounds).
 
+1. Rescan SSH keys on all storage nodes
```

Comment:
Resets ssh keys for storage nodes. These change during the upgrade procedure and result in noisy invalid ssh key errors.